src/components/NavBar.vue

```diff
@@ -117,46 +108,49 @@
     display: block;
   }
 
-  .navbar-nav>li > a > span{
+  .navbar-nav > li > a > span {
     color: white;
     font-size: 1.6rem;
   }
-  .navbar-nav>li > a > span:hover{
+
+  .navbar-nav > li > a > span:hover {
     color: orange;
   }
-  .container-fluid{
+
+  .container-fluid {
     margin-left: 15%;
```

Comment:
Let margin-left equals to margin-right? Otherwise the navbar is quite strange, there is some extra space before IoTDB logo.

Reply:
thanks for your review, i have changed this.